core/src/commonMain/kotlin/org/aquila3d/core/math/Vector3.kt

```diff
@@ -157,6 +157,9 @@ class Vector3 {
         @JvmStatic
         fun NEG_Z(): Vector3 = Z().invert()
 
+        @JvmStatic
+        fun ONES(): Vector3 = Vector3(1.0, 1.0, 1.0)
```

Comment:
This looks fine but do `cross` and `dot` need to create new Vector3 instances to perform the math given they're returning new instances as an output?

Reply:
These companion object/static ones are intended specifically to create new instances rather than do the math in place like the member versions.